cmiles/tests/test_cmiles.py

```diff
@@ -379,3 +379,36 @@ def test_input_mapped():
     mol_2 = cmiles.utils.load_molecule(mol_id['canonical_isomeric_explicit_hydrogen_mapped_smiles'], backend='rdkit')
     assert cmiles.utils.is_mapped(mol_1, backend='rdkit') == False
     assert cmiles.utils.is_mapped(mol_2, backend='rdkit') == True
+
+
+@using_openeye
+def test_chemical_formula_oe():
```

Comment:
Might be good to test Lithium oxide (`LiO2`) which will test `ncount == 1` and capitol case. Not necessarily saying to do this here, but if you have different backend functions you can do something like to explore all options:

```python
def _openeye_from_smiles(smiles):
     molecule = cmiles.utils.load_molecule(smiles, backend='openeye')
     oechem.OEAddExplicitHydrogens(molecule)
     return cmiles.generator.molecular_formula(molecule, backend='openeye')

@pytest.parametrize("backend", [_openeye_from_smiles, _rdkit_from_smiles])
@pytest.parametrize("smiles, bench", [("CCCC", "C4H10"), ("LiO2", "LiO2")])
def test_molecule_formula(backend, smiles, bench):
    assert backend(smiles) == bench
```

A design template to consider at least for these type of tests.

Reply:
When using this design template, how do I skip only one or the other if openeye or rdkit are not installed? I test different environments where only one is installed.

Reply:
Figured it out. Thanks!

Reply:
Actually, my solution here is not working. 
I don't really have to test it separately so I can skip unless both dependencies are installed.

Reply:
Yea, you would need to do something like `pytest.importorskip("openeye")`. `pytest.mark.skip` cannot happen *inside* of a test, you need to use either the above or `pytest.skip(reason)` *inside* of a test.